### ISSUE TITLE
Differentiating faceted accordion lists terms and their search links (from 2060)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#95]
**Problem Description:**
In the `Related Materials` accordion there are links listed on the right and left hand side.  
For ex:
"laid paper"....... "Show all 27 results"
Clicking on the term will bring you to the page for laid paper while clicking on the "show all" will lead to all results for laid paper + your original search (the intersection of the two). 

This is confusing to users as the links lead to different search results while containing names/labels that do not indicate the two are different. 
 
**Expected Behavior/Solution:**
Add  new label to the top of each column to indicate a difference between the links. 

**Requirements:**
- The links on the left  will contain the bolded label `Concept` as it's header. 
- The links on the right will contain the bolded label `Related results` as it's header. 
- Links will continue working as expected. 
- All requirements for desktop must be included for the mobile version. (mockup needed) 

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] ~POW design consideration~
- [x] Wireframe/Mockup (rough sketch below) - Mike
- [x] Committee discussions - Sarah
- [x] Feasibility/Team discussion - Sarah
- [x] Backend requirements - TBD
- [x] Frontend requirements- TBD
- [x] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**
- https://lux-front-tst.collections.yale.edu/view/person/2c4f7220-ac16-4017-be21-c8aac0478f9a 

**Dependencies/Blocks:**
N/A

**Related Github Issues:**
N/A

**Related links:**
- https://www.bugherd.com/projects/284041/tasks/2003
- POW Designs (Slide 53-55): https://drive.google.com/file/d/1nymAqJ33CRFPHYSNkqkJt6WvJkxixRdC/view?usp=drive_link

**Wireframe/Mockup:**
Current: 
![image](https://github.com/project-lux/lux-frontend/assets/160149818/3ab27945-e4ec-4567-880e-06425391c799)

Expected: 
![Image](https://github.com/user-attachments/assets/c01a2ed0-e8c7-458a-9967-02b9d2cdc870)
_Mobile_

![Image](https://github.com/user-attachments/assets/270f45c0-6f07-4ad0-9b59-ea7d7bca801f)

**Comment by prowns** [comment]
Issue currently shared with POW.

**Comment by roamye** [comment]
Per ba/ux meting we need an updated mock based on the sketch listed under Wireframe/Mockup.

**Comment by roamye** [comment]
Do we want the second line to be indented? or to display right below it as it does in related concepts? Wanted to make sure I was going in the right direction before continuing. The mockup below shows it as right below. 

![Image](https://github.com/project-lux/lux-frontend/assets/163148482/400b43a5-0a94-4887-b07f-b864ee0f6b62)

**Comment by roamye** [comment]
updated wireframe with an indent

**Comment by kamerynB** [comment]
@roamye @miketullo95 @prowns These wireframes are not valid. This is a faceted search and we do not have this type of granularity to add an additional label describing the relationship of the material to the entity. It would have to be changed to a related list search.

**Comment by prowns** [comment]
Per 5/28 team meeting, this will impact all faceted lists.  Needs further discussion with @kamerynB involved.  Team questioning why this needs to be changed.  

alternate ideas

- “Show all N related results”
- Add related before results?
- change the label on “Show all N results” somehow

**Comment by prowns** [comment]
@miketullo95 - I think this ticket is with you for UX consideration.

**Comment by roamye** [comment]
from ux/ba 7/16: 
we will wait for mike to get back before moving forward with this issue. lower priority for heather.

**Comment by kamerynB** [comment]
@roamye Do we want to move this issue to the "Future Work" column since we intend on waiting for Mike to return from his leave?

**Comment by prowns** [comment]
Forming and blocked by Mike's leave.

**Comment by miketullo95** [comment]
catching up on this, @prowns @roamye whats the next steps on this, are we okay with going with the direction of the latest wireframes? ![Image](https://github.com/user-attachments/assets/2838610b-1d62-4c92-bcc1-97ad316bcc56)

**Comment by roamye** [comment]
@miketullo95 - from what I remember, the team was not in favor of the wireframes in the above comment. There were alternate ideas listed in this comment [here](https://github.com/project-lux/lux-frontend/issues/95#issuecomment-2127837005) but ultimately the new wireframe are up to your descrestion. Given you have a UX perspective, I wonder if you had an idea on a better design for this?

**Comment by roamye** [comment]
@miketullo95 - added the POW designs link in the into the requirements as an extra resource. Although, that is not the route we will want to take. 

Proposal: Add a header to each side of the columns. 
Ex: 
CONCEPTS  | RELATED RESULTS
paper          | Show all 23 objects 
ink               | Show all 4 objects

CONCEPST label can be "material related to this person or group"

**Comment by miketullo95** [comment]
@roamye wireframe uploaded. I was contemplating also adding a divider to represent the two columns more, but decided not to and went with a simpler approach.

**Comment by roamye** [comment]
@miketullo95 - the wireframe was approved by UAT yesterday. Re-formatted the issue to match the wireframe.

**Comment by roamye** [comment]
@miketullo95 - new mockup is needed for how this will look on mobile.

**Comment by roamye** [comment]
Approved by UAT

**Comment by roamye** [comment]
LG in PRD closing.
